spec/models/shopping_cart_item_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe ShoppingCartItem, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :owner_id }
+    it { is_expected.to have_db_column :owner_type }
+    it { is_expected.to have_db_column :quantity }
+    it { is_expected.to have_db_column :item_id }
+    it { is_expected.to have_db_column :item_type }
+    it { is_expected.to have_db_column :price_cents }
+    it { is_expected.to have_db_column :price_currency }
+  end
+
+  it 'should have a valid factory' do
+    expect(create(:shopping_cart_item)).to be_valid
+  end
+end
```

Comment:
I think you should add a spec for `belongs_to` and any validations (if there are any)